processor/groupbyattrsprocessor/README.md

```diff
@@ -0,0 +1,36 @@
+# Group by Attributes processor
+
+Supported pipeline types: traces, logs
+Status: in development
+
+This processor groups the records by provided attributes, extracting them from the 
+record to resource level. 
+
+Typical use-cases:
+* extracting resources from "flat" data formats, such as Fluentbit logs
+* optimizing data by extracting common attributes
```

Comment:
https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md032---lists-should-be-surrounded-by-blank-lines

> MD032/blanks-around-lists: Lists should be surrounded by blank linesmarkdownlintMD032


```suggestion
Typical use-cases:

* extracting resources from "flat" data formats, such as Fluentbit logs
* optimizing data by extracting common attributes
```